build.gradle

```diff
@@ -67,7 +67,7 @@ dependencies {
 }
 
 shadowJar {
-    archiveFileName = 'addressbook.jar'
+    archiveFileName = 'sportsync.jar'
```

Comment:
Great observation. Good to change all occurrences of addressbook to SportSync to personalise our product